.sequelizerc

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
+
+if (process.env != 'production'){
+	module.exports = {
+	  "config": path.resolve('./server/src/config', 'config.js'),
+	  "models-path": path.resolve('./server/src/models'),
+	  "seeders-path": path.resolve('./server/src/seeders'),
+	  "migrations-path": path.resolve('./server/src/migrations')
+	};	
+}else{
+	module.exports = {
+  "config": path.resolve('./server/dist/config', 'config.js'),
+  "models-path": path.resolve('./server/dist/models'),
+  "seeders-path": path.resolve('./server/dist/seeders'),
+  "migrations-path": path.resolve('./server/dist/migrations')
+	};
+}
```

Comment:
Is this really necessary? Also, your indentation is off. Would love to hear from you as to your reason for separating this